src/vos/README.md

```diff
@@ -472,34 +472,13 @@ A special aggregation ULT processes aggregation, yielding frequently to avoid bl
 
 <a id="79"></a>
 
-## VOS Checksum Management (to be updated)
-
-One of the guarantees that VOS provides is end-to-end data integrity.
-Data corruption in VOS can happen while reading or writing data due to a various
-reasons, including leaks or failures in persistent memory, or during data
-transmission through the wire. VOS supports data integrity check with checksums.
-
-Client side (HDF5, DAOS-SR, and DAOS) of the stack provides checksum for both
-byte extents and key value stores on writes and updates, respectively.
-
-The VOS API for updates and writes will require checksums as arguments from its
-upper layer(s). VOS requires checksum for both keys and values in case of KV
-objects.  VOS stores the checksum along with the data.
-
-A Lookup operation on a KV will verify the checksum by computing the checksum
-for the key and value. If reads in byte-arrays spans over multiple extent
-ranges, VOS would have to recompute the checksum at the server for each
-individual extent range for verifying their integrity and return the computed
-checksum of the entire requested extent range to the client.  In case if a read
-requests a partial byte array extent of an existing extent range, VOS would
-compute the checksum of the existing extent to verify correctness and then
-return the requested extent range to the client with its computed checksum.
-When byte array extents are aggregated, VOS individually re-computes checksum of
-all extent ranges to be merged to verify correctness, and finally computes and saves the checksum for the merged extent range.
-
-Because checksum computation involves multiple layers of stack to be in-sync, VOS plans to leverage and extend either the mchecksum library or the
-<a href="https://01.org/intel&#174;-storage-acceleration-library-open-source-version">
-Intel&#174; Storage acceleration library</a>.
+## VOS Checksum Management (TODO)
+
+One of the guarantees that VOS provides is end-to-end data integrity.  Data corruption in VOS can happen while reading or writing data due to a various reasons, including leaks or failures in persistent memory, or during data transmission through the wire. VOS supports data integrity check with checksums.
```

Comment:
- due to various reason, or due to a variety of reasons/causes
- do we need check after data integrity?